.github/workflows/release.yml

```diff
@@ -25,6 +25,12 @@ jobs:
           NPM_EMAIL: ${{secrets.NPM_EMAIL}}
           NPM_USERNAME: ${{secrets.NPM_USERNAME}}
         run: yarn release
+      - name: Set up git identity
+        run: git config --global user.name "instructure-ui-ci" && git config --global user.email "instructure-ui-ci@instructure.com"
+      - name: Add tag
+        run: git tag -a v$(./.github/scripts/calculateVersion.sh) -m v$(./.github/scripts/calculateVersion.sh)
```

Comment:
I'd add here a comment why we use the -a and -m flags

Reply:
Done